front/lib/resources/group_resource.ts

```diff
@@ -371,14 +371,25 @@ export class GroupResource extends BaseResource<GroupModel> {
 
   async getActiveMembers(auth: Authenticator): Promise<UserResource[]> {
     const owner = auth.getNonNullableWorkspace();
-    const memberships = await GroupMembershipModel.findAll({
-      where: {
-        groupId: this.id,
-        workspaceId: owner.id,
-        startAt: { [Op.lte]: new Date() },
-        [Op.or]: [{ endAt: null }, { endAt: { [Op.gt]: new Date() } }],
-      },
-    });
+
+    let memberships: GroupMembershipModel[] | MembershipResource[];
+
+    // The global group does not have a DB entry for each workspace member.
+    // TODO(GROUPS_INFRA): Remove this once we consolidate memberships with group memberships.
```

Comment:
What's the rationale for creating these entries?

Reply:
Creating which entries? The groups memberships? If yes, we discussed that we could phase out the regular memberships table in favor of the group_memberships. Obviously, this is not part of the current milestone. Today, no one has a membership on the global group, but this abstraction should not be spread in the front end and the server should still return the whole list.

Reply:
ack